src/main/java/ch/njol/unofficialmonumentamod/mixins/ThreadExecutorMixin.java

```diff
@@ -12,13 +14,15 @@
 @Mixin(ThreadExecutor.class)
 public abstract class ThreadExecutorMixin<R extends Runnable> {
 
+	Pattern teamRemovalPattern = Pattern.compile("Player is either on another team or not on any team. Cannot remove from team '\\w+'\\.");
+
 	/**
 	 * Silences errors that happen when a player leaves monumenta or changes shard.
 	 */
 	@Inject(method = "executeTask", at = @At(value = "INVOKE", target = "Lorg/slf4j/Logger;error(Lorg/slf4j/Marker;Ljava/lang/String;Ljava/lang/Object;Ljava/lang/Object;)V", remap = false), locals = LocalCapture.CAPTURE_FAILSOFT, cancellable = true)
 	private void silenceTeamFatal(R task, CallbackInfo ci, Exception exception) {
-		if (UnofficialMonumentaModClient.options.silenceTeamErrors) {
-			if (((exception.getMessage().matches("Player is either on another team or not on any team. Cannot remove from team '\\w+'\\.")|| Objects.equals(exception.getMessage(), "Cannot invoke \"net.minecraft.scoreboard.Team.getName()\" because \"team\" is null")))) {
+		if (UnofficialMonumentaModClient.options.silenceTeamErrors && exception != null) {
```

Comment:
The null check should test `exception.getMessage()` - although a null test for the exception is probably not a bad idea either, so probably keep that is as well.